cpp/arcticdb/storage/library_manager.cpp

```diff
@@ -115,14 +119,61 @@ void LibraryManager::write_library_config(const py::object& lib_cfg, const Libra
     );
 }
 
+void LibraryManager::modify_library_option(const arcticdb::storage::LibraryPath &path,
+                                           std::variant<ModifiableLibraryOption, ModifiableEnterpriseLibraryOption> option,
+                                           std::variant<bool, int64_t> new_value) const {
```

Comment:
A type alias for `std::variant<bool, int64_t>` will help

Reply:
Thanks, fixed.